server/pa.js

```diff
@@ -209,6 +209,32 @@ async function handleMessageClients(req, res) {
   }
 }
 
+const validateCheckDatabaseConnection = Validator.body(['braveKey', 'googleIdToken']).trim().notEmpty()
+
+async function handleCheckDatabaseConnection(req, res) {
+  const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
+
+  if (validationErrors.isEmpty()) {
+    const braveAPIKey = req.body.braveKey
+
+    if (paApiKeys.includes(braveAPIKey)) {
+      try {
+        await db.getCurrentTime()
+        res.status(200).send({ message: 'success' })
+      } catch (err) {
+        helpers.logError(err)
+        res.status(500).send({ message: 'Error in Database Access' })
```

Comment:
I wonder if we want a Sentry message here too?

Reply:
I believe `helpers.logError(err)` does log to Sentry. Unless you're talking about a different line?

Reply:
Ahh that's a good call - I will add it!

Reply:
Yes, you're right, it does log to Sentry too!

Reply:
Whoops I second guessed myself and deleted my comment about `helpers.logError()` including a sentry log. I will leave as is.